kanvas/src/commonMain/kotlin/svg/SvgKanvas.kt

```diff
@@ -7,18 +7,21 @@ import com.juul.krayon.kanvas.Paint
 import com.juul.krayon.kanvas.PathBuilder
 import com.juul.krayon.kanvas.Transform
 import com.juul.krayon.kanvas.split
+import com.juul.krayon.kanvas.xml.NumberFormatter
+import com.juul.krayon.kanvas.xml.ToStringFormatter
 import com.juul.krayon.kanvas.xml.XmlElement
 import com.juul.krayon.kanvas.xml.escape
 
 public class SvgKanvas(
     override val width: Float,
     override val height: Float,
+    private val formatter: NumberFormatter = ToStringFormatter()
```

Comment:
I like the choice of having the faster formatter be default, then consumers can choose the deterministic formatter (and we can use it for unit tests). Awesome!